content/en/integrations/faq/powershell-command-to-install-azure-datadog-extension.md

```diff
@@ -76,18 +76,31 @@ The normal settings include:
 |----------|------|--------------|
 | `site` | String | Set the Datadog intake site, for example: `SITE=`{{< region-param key="dd_site" code="true">}} |
 | `agentVersion` | String | The Agent version to install, following the format `x.y.z` or `latest` |
+| `agentConfiguration` | URI | (optional) Url to the Azure blob contaning the Agent configuration as a zip. |
+| `agentConfigurationChecksum` | String | The SHA256 checksum of the Agent configuration zip file, mandatory if `agentConfiguration` is specified. |
```

Comment:
By "zip file" do you mean any compressed file type, or is the configuration zip file a specific type?

Reply:
Specifically a zip file type.